newIDE/app/src/SceneEditor/index.js

```diff
@@ -445,16 +453,22 @@ export default class SceneEditor extends React.Component<Props, State> {
     );
   };
 
-  _onObjectSelected = (selectedObjectName: string) => {
-    if (!selectedObjectName) {
-      this.setState({
-        selectedObjectNames: [],
-      });
-    } else {
-      this.setState({
-        selectedObjectNames: [selectedObjectName],
-      });
+  _onObjectSelected = (objectWithContext: ?ObjectWithContext = null) => {
+    const selectedObjectNames = [];
+    if (objectWithContext) {
+      selectedObjectNames.push(objectWithContext.object.getName());
```

Comment:
same here, the logic changes to adding an additional object to the selection.
Is that on purpose?

Reply:
edit: same here, sorry :)